apps/studio/components/ui/SchemaSelector.tsx

```diff
@@ -106,7 +110,7 @@ const SchemaSelector = ({
               </div>
             </Button>
           </PopoverTrigger_Shadcn_>
-          <PopoverContent_Shadcn_ className="p-0 w-64" side="bottom" align="start">
+          <PopoverContent_Shadcn_ className="p-0 popover-trigger-width" side="bottom" align="start">
```

Comment:
should we bake the `popover-trigger-width` into the PopoverContent_Shadcn component instead? at the top of my head feels like this is gonna be needed everywhere?